src/v2/Apps/Home/Components/HomeCurrentFairsRail.tsx

```diff
@@ -0,0 +1,182 @@
+import {
+  Box,
+  Image,
+  Text,
+  Flex,
+  Spacer,
+  Shelf,
+  Skeleton,
+  SkeletonText,
+  SkeletonBox,
+} from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/System"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { HomeCurrentFairsRail_viewer } from "v2/__generated__/HomeCurrentFairsRail_viewer.graphql"
+import { HomeCurrentFairsRailQuery } from "v2/__generated__/HomeCurrentFairsRailQuery.graphql"
+
+interface HomeCurrentFairsRailProps {
+  viewer: HomeCurrentFairsRail_viewer
+}
+
+const HomeCurrentFairsRail: React.FC<HomeCurrentFairsRailProps> = ({
+  viewer,
+}) => {
+  if (viewer.fairs?.length === 0) {
+    return null
+  }
+
+  return (
+    <HomeCurrentFairsContainer>
+      <Shelf alignItems="flex-start">
+        {viewer.fairs!.map((fair, index) => {
+          if (!fair) {
+            return <></>
+          }
+
+          return (
+            <RouterLink
+              to={`/fair${fair.href}`}
+              textDecoration="none"
+              key={index}
+            >
+              <Box key={index}>
+                {fair.image?.cropped?.src && (
+                  <Image
+                    src={fair.image.cropped.src}
+                    srcSet={fair.image.cropped.srcSet}
+                    width={fair.image.cropped.width}
+                    height={fair.image.cropped.height}
+                    lazyLoad
+                  />
+                )}
```

Comment:
I noticed [this in the featured galleries rail](https://artsyproduct.atlassian.net/browse/GRO-534) — these should also have a fallback box in the event there's no image.